README.md

```diff
@@ -39,6 +40,11 @@ services:
         ENTRYPOINT: ./YOUR_ENTRYPOINT
 ```
 
+By default, the IOC will be built with statically linked libraries. If you want
+to use dynamically linked libraries, you should define the build target as
```

Comment:
"EPICS libraries" instead of "libraries".

It's "if you *need* to use", not "want" (use bold, even)